app/views/spree/admin/orders/_note.html.haml

```diff
@@ -3,7 +3,7 @@
     %td{ colspan: "5", data: { controller: "input-char-count" }, style: "position: relative;" }
       %label
         = t(".note_label")
-      = text_field_tag :note, @order.note, { maxLength: 280, data: { "input-char-count-target": "input" } }
+        = text_field_tag :note, @order.note, { maxLength: 280, data: { "input-char-count-target": "input" } }
```

Comment:
Is it voluntary that you've put the `text_field` into the `label` tag? I'm wondering why you did this, even if it seems relevant in certains cases for sure (https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly)

Reply:
Honestly, I thought you'd meant to put the `text_field` into the `label` tag but somehow it got screwed up. That's why I put it inside the label tag. I just thought it was good practice to have labels associated with input elements, no? :shrug:

Reply:
Great. This is useful to associate the textfield with the label without using ids. I think it's good practice to always have either: put the input inside the label or use an id to associate the two.